holdingsiq.json

```diff
@@ -2226,6 +2218,477 @@
+                  "holdingsevent-filter": {
+                    "value": {
+                      "created": "2023-02-01T22:32:33.653Z",
+                      "custId": "ns334455",
+                      "fileName": "FileName",
+                      "format": "TSV",
+                      "holdingsEvent": {
+                        "endDate": "09-01-2022",
+                        "eventActions": "[\"IGNORED\"]",
+                        "eventSource": "[\"SSD\"]",
+                        "startDate": "08-31-2022"
+                      },
+                      "id": "9cc047c6-605f-40f5-be64-7f638124bcfd",
+                      "status": "QUEUED",
+                      "type": "HoldingsEvent"
+                    }
+                  },
+                  "cost-filter": {
+                    "value": {
+                      "costDownload": {
+                        "currency": "USD",
+                        "fromFiscalYear": 2020,
+                        "toFiscalYear": 2022
+                      },
+                      "created": "2023-02-01T22:32:33.653Z",
+                      "custId": "ns334455",
+                      "fileName": "FileName",
+                      "format": "TSV",
+                      "id": "9cc047c6-605f-40f5-be64-7f638124bcfd",
+                      "status": "QUEUED",
+                      "type": "COST"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Unexpected Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/errors"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/{custid}/exports/{exportid}": {
+      "get": {
+        "tags": [
+          "Holdings Resources"
+        ],
+        "summary": "Get Export Export Id",
+        "description": "This endpoint allows customers to retrieve details about a specific export request of their holdings, including an array of objects that represent metadata about the single export. Parameters include a creation date, export ID, status, type, size, and links. The type of export can be one of KBART2, STANDARD, COST, or HOLDINGSEVENT, while the status can be one of QUEUED, INPROGRESS, or COMPLETED. The links array contains objects that specify a presigned URL to download the export file from, which is valid for 30 minutes. If the URL expires, a new URL can be requested from the API. \n\n  **Please Note:**  Your production API key to access HoldingsIQ will not give you access to our sandbox. Therefore, your production API key will not give you access to the API through our interactive documentation. You will need a sandbox API key to gain access to the API through our interactive documentation or to use the sandbox directly. If you require a sandbox API key, please contact EBSCO customer support.",
```

Comment:
This endpoint allows customers to retrieve details about a specific export request of their holdings, including an array of objects that represent metadata about the single export. Parameters include a creation date, export ID, status, type, size, and links. The type of export can be one of KBART2, STANDARD, COST, or HOLDINGSEVENT.  The status can be one of QUEUED, INPROGRESS, or COMPLETED. The links array contains objects that specify a presigned URL to download the export file from.   The URL is valid for 30 minutes. If the URL expires, a new URL can be requested from the API. \n\n **Please Note:** Your production API key to access HoldingsIQ will not give you access to our sandbox. Therefore, your production API key will not give you access to the API through our interactive documentation. You will need a sandbox API key to gain access to the API through our interactive documentation or to use the sandbox directly. If you require a sandbox API key, please contact EBSCO customer support.